stories/react/PopupMenu.stories.js

```diff
@@ -0,0 +1,37 @@
+
+import React from 'react';
+import Examples from './utils/Examples.js';
+import PopupMenu from '../../src/react/PopupMenu.js';
+import PopupMenuItem from '../../src/react/PopupMenuItem.js';
+import HTMLPopupMenu from '../../components/menu/popup-menu.html';
+import HTMLPopupMenuRelative from '../../components/menu/relative-popup-menu.html';
+
+let examples = {
+	'Basic popup menu (static)': {
+		jsx: <PopupMenu onMenuItemClick={() => {}}>
+				<PopupMenuItem key='1' itemId='1' value='item 1 (selected)' selected/>
```

Comment:
Why not use `PopupMenu.Item` ?

Reply:
We've also thought about this. We went this way in the modal. In general, we would like to expose all components from lib/react, so we export them all individually.